lein-eftest/src/leiningen/eftest.clj

```diff
@@ -60,10 +68,22 @@
                           ~selectors)]
       ns#)))
 
+(defn- process-options
+  [{:keys [report report-to-file] :as options}]
+  ;; Use `or` as opposed to on the keyword's IFn to ensure we override an
+  ;; explicit/accidental `nil`.
+  ;;
+  ;; Note, that we've duplicated the default reporter decision both here and in
+  ;; `eftest.runner/run-tests`.
+  (let [reporter (or report eftest.report.progress/report)]
+    (cond-> (or options {})
+      (some? report-to-file)
+      (update :report eftest.runner/report-to-file report-to-file))))
```

Comment:
I could return `nil` rather than an empty map if there's nothing in the `project.clj` file, but figured the result of processing things could/should be an empty map rather than `nil`. Happy to change.